Makefile

```diff
@@ -123,6 +123,11 @@ $(EASYJSON_BIN): deps
 	$(SILENT)echo "+ $@"
 	go install github.com/mailru/easyjson/easyjson
 
+CONTROLLER_GEN_BIN := $(GOBIN)/controller-gen
+$(CONTROLLER_GEN_BIN): deps
+	$(SILENT)echo "+ $@"
+	go install sigs.k8s.io/controller-tools/cmd/controller-gen
```

Comment:
I think we should pin the version. Or does it take the version from go.mod?

Reply:
It takes version from go.mod. I've created https://github.com/stackrox/stackrox/pull/3064 to move generators to separated go module like linters so they code be independent from our other deps.